src/helper/jokes.ts

```diff
@@ -46,6 +46,12 @@ export async function handleJokes(
   }
 }
 
+async function extractMeme(url: string) {
+  const { data } = await axios.get(url);
+  const children = data.data.children.filter((child: any) => child.data.url.endsWith(".jpg") || child.data.url.endsWith(".png"));
```

Comment:
optional chaining would be safer. data?.data?.children?. and so on

Reply:
Any way to test these changes? I doubt if I could